Example/Tests/CISAdAnalyticsTestDouble.swift

```diff
@@ -76,9 +76,15 @@ class CISAdAnalyticsTestDouble: NSObject, CISAdAnalyticsProtocol, TestDoubleData
     }
 
     func reportAdMetric(_ key: String, value: Any) {
+        let valueToTrack: String
+        if let size = value as? CGSize {
+            valueToTrack = "\(size.width)x\(size.height)"
```

Comment:
This is a workaround for testing as printing a `CGSize` value can have 2 different formats somehow.